### DISCUSSION TITLE
update to gpt-4o

### DISCUSSION THREAD

**Discussion by PhilBrk8** [wasp-lang/wasp#2042] [Archived]
could you update the mage to utilize the gpt-4o model?
As of today, gpt-4-0613 is in use and gpt-4o would provide faster and cheaper API-use

**Comment by Martinsos** [comment]
That is a good idea and we will be looking into it! It requires some testing from our side, as new models can sometimes surprisingly provide worse results. I think gpt-4o should be an improvement though from what I read online, but still, we need to do it properly. I will convert this into an issue so we can get to it soon!

**Comment by Martinsos** [comment]
Ok, here is the issue: https://github.com/wasp-lang/wasp/issues/2043 .
